drivers/ethernet/phy/phy_dm8806.c

```diff
@@ -0,0 +1,276 @@
+/* DM8806 Stand-alone Ethernet PHY with RMII
+ *
+ * Copyright (c) 2024 Indurad GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT davicom_dm8806_phy
+
+#define LOG_MODULE_NAME eth_dm8806_phy
+#define LOG_LEVEL       CONFIG_ETHERNET_LOG_LEVEL
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
```

Comment:
you don't need to define macros for the arguments, also this macro has a form that can take both arguments

Reply:
Useless definition are removed